app/views/locations/index.html.erb

```diff
@@ -1,9 +1,27 @@
-<ol>
+<% content_for(:page_title, "Face-to-face appointments near #{@postcode}  - Pension Wise") %>
+
+<h1>Face-to-face appointments near <%= @postcode %></h1>
```

Comment:
Should we tidy up the user input? If I've entered a lowercase postcode should we show that or should we uppercase it?